editor/scene/texture/texture_editor_plugin.cpp

```diff
@@ -271,6 +298,22 @@ TexturePreview::TexturePreview(Ref<Texture2D> p_texture, bool p_show_metadata) {
 	const Image::Format format = p_texture.is_valid() ? get_texture_2d_format(p_texture.ptr()) : Image::FORMAT_MAX;
 	const uint32_t components_mask = format != Image::FORMAT_MAX ? Image::get_format_component_mask(format) : 0xf;
 
+	// Setup Mipmap selector
+	const int mipmaps = get_texture_mipmaps_count(p_texture);
+	if (mipmaps > 0) {
+		mipmap_spinbox = memnew(SpinBox);
+		mipmap_spinbox->set_tooltip_text("Mipmap level viewer.");
```

Comment:
```suggestion
		mipmap_spinbox->set_tooltip_text(TTRC("Mipmap level viewer."));
```
Also I'd change the text to something like `Mipmap index.`. It's odd calling SpinBox a "viewer".